server/src/models/noncommercial-application.es6

```diff
@@ -254,6 +254,11 @@ module.exports = sequelize.define(
       field: 'signature',
       allowNull: false
     },
+    authEmail: {
```

Comment:
Is authEmail the user's email or the authorizing official's email? would it make sense to disambiguate this fieldname?